UICatalog/Scenarios/ProgressBarStyles.cs

```diff
@@ -88,52 +86,41 @@ public override void Setup ()
 					}, null, 0, _timerTick);
 				}
 			};
-			leftContainer.Add (button);
-			Win.Add (leftContainer);
 
-			var rightContainer = new View () {
-				X = Pos.Right (leftContainer),
-				Y = Pos.Bottom (ckbBidirectional) + 1,
-				Width = Dim.Fill (),
-				Height = Dim.Fill ()
-			};
```

Comment:
ou need to add these containers, otherwise the framed progress bar will not be visible.

Reply:
Now I see the really changes. But see my comment here https://github.com/BDisp/Terminal.Gui/pull/168#issuecomment-1813484321.
Please resolve the conflicts. Thanks.